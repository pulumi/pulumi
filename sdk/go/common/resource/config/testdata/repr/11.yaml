my:mapKey:
  nums:
  - 1
  - 2
  - 3
