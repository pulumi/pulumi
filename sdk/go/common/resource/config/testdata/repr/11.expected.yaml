decrypt:
  my:mapKey: '{"nums":[1,2,3]}'
paths:
  my:mapKey:
    value:
      nums:
      - 1
      - 2
      - 3
    string: '{"nums":[1,2,3]}'
    redacted: '{"nums":[1,2,3]}'
    object: KBAAF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9/4EEAQL/ggABDAEQAABJ/4IiAAEEbnVtcw5bXWludGVyZmFjZSB7ff+DAgEC/4QAARAAACP/hCAAAwVpbnQ2NAQCAAIFaW50NjQEAgAEBWludDY0BAIABg==
    secure: false
    isObject: true
  my:mapKey.nums:
    value:
    - 1
    - 2
    - 3
    string: '[1,2,3]'
    redacted: '[1,2,3]'
    object: HRAADltdaW50ZXJmYWNlIHt9/4MCAQL/hAABEAAAI/+EIAADBWludDY0BAIAAgVpbnQ2NAQCAAQFaW50NjQEAgAG
    secure: false
    isObject: true
  my:mapKey.nums[0]:
    value: "1"
    string: "1"
    redacted: "1"
    object: DhAABnN0cmluZwwDAAEx
    secure: false
    isObject: false
  my:mapKey.nums[1]:
    value: "2"
    string: "2"
    redacted: "2"
    object: DhAABnN0cmluZwwDAAEy
    secure: false
    isObject: false
  my:mapKey.nums[2]:
    value: "3"
    string: "3"
    redacted: "3"
    object: DhAABnN0cmluZwwDAAEz
    secure: false
    isObject: false
